from domain.errors import DomainError


class IdentificationNotFoundError(DomainError):
    pass
